Source/ScreenRotate.spoon/init.lua

```diff
@@ -0,0 +1,113 @@
+--- === ScreenRotate ===
+---
+--- Toggle rotation on external screens
+---
+--- Download: [https://github.com/Hammerspoon/Spoons/raw/master/Spoons/ScreenRotate.spoon.zip](https://github.com/Hammerspoon/Spoons/raw/master/Spoons/ScreenRotate.spoon.zip)
+---
+--- Makes the following simplifying assumptions:
+--- * That you only toggle between two positions for rotated/not
+---   rotated (configured in `rotating_angles`, and which apply to all
+---   screens)
+--- * That "rotated" means "taller than wider", for the purposes of
+---   determining if the screen is rotated upon initialization.
+
+local obj={}
+obj.__index = obj
+
+-- Metadata
+obj.name = "ScreenRotate"
+obj.version = "0.1"
+obj.author = "Diego Zamboni <diego@zzamboni.org>"
+obj.homepage = "https://github.com/Hammerspoon/Spoons"
+obj.license = "MIT - https://opensource.org/licenses/MIT"
+
+-- Spoon self.logger
+obj.logger = hs.logger.new('ScreenRotate')
+
+--- ScreenRotate.screens_to_skip
+--- Variable
+--- Lua patterns for screens that shouldn't be rotated, even if they match one of the patterns. Defaults to `{ "Color LCD" }`, which excludes the built-in display on a laptop.
+obj.screens_to_skip = { "Color LCD" }
```

Comment:
You might also want to include `"iMac"` here, which is what the internal LCD is called on iMacs (which are specifically not supposed to be rotated, per Apple's hardware manuals)